packages/google-tag-manager/index.js

```diff
@@ -46,6 +48,19 @@ module.exports = async function nuxtTagManager(_options) {
     async: true
   })
 
+  // prepend google tag manager <noscript> fallback to <body>
+  this.options.head.noscript.push({
```

Comment:
I will actually add a sanitization before to avoid some error like cannot push to `undefined`:

```js
this.options.head.noscript = this.options.head.noscript || []
```

Reply:
Same above for `this.options.head.script` (and also `this.options.head = this.options.head || {}`)

Reply:
Thanks for the input :-)